### PR TITLE
hello world project to assess tools

### PR DESCRIPTION
**Pull request by basem909**
this is the hello world project to test all the setups and linters to make sure they are ready
there is an index.html file with its stylesheet and the linters file for both HTML and css

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# hello-world
\ No newline at end of file
+hello world
+the target of this project is to get sure about the tools that will be used through the course
+
+Built With
+html and css
+
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly make your readme file more descriptive by using the [microverse readme template](https://github.com/microverseinc/readme-template) this make your project more descriptive and professional 👍🏿

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1{
+    color: aquamarine;
+}
```
Comment (prolajumokeoni) [inline styles.css]: - [ ] Kindly link your css file to html file so you can see styles displayed on your browser

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @basem909 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good PR Title and description

## Required Changes ♻️
- [ ] Kindly set up [Github actions](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-github-actions) in your project using the  microverse linter config files

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving  your  branch a more descriptive name that describes what the branch entails, examples of good titles are:  setup-project, build-navbar


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (BenMukebo) [reply styles.css]: > * [ ]  Kindly link your CSS file to an HTML file so you can see styles displayed on your browser

- [OPTIOANL ] Great job on implementing this change as required, you have linked your CSS file with the HTML but I think you have forgotten to rename the file path, I think you have changed the style.css to stylesheet1.css, fix that as you go to the next project.

**Review by BenMukebo — APPROVED** [review]
## Hi @basem909 , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**